app/gui2/src/bindings.ts

```diff
@@ -1,9 +1,20 @@
-import { defineKeybinds } from '@/util/shortcuts'
+import { defineKeybinds, keybindMixin } from '@/util/shortcuts'
+
+// Undo/redo may require different handlers in different contexts, but should have the same default bindings everywhere.
+const undoRedoBindings = keybindMixin({
+  undo: ['Mod+Z'],
+  redo: ['Mod+Y', 'Mod+Shift+Z'],
+})
```

Comment:
If I understand correctly, the key bind sheet editable by user will allow setting a different "undo" shortcut for every panel. I'm not sure if it's desirable. Perhaps `undoRedoBindings` could be created using `defineKeybinds` as usual, and every panel may create a handler from it, and attach it to a proper event listener next to its "main" handler.